src/components/ui/OurMission.tsx

```diff
@@ -0,0 +1,106 @@
+// src/components/ui/OurMission.tsx
+
+import React from 'react';
```

Comment:
```suggestion
import React from 'react';
```
React's importation is not necessary, please remove